src/lem/store.rs

```diff
@@ -1063,11 +1063,9 @@ impl<F: LurkField> Store<F> {
         let mut stack = vec![ptr];
         macro_rules! feed_loop {
             ($x:expr) => {
-                if $x.is_hash() {
```

Comment:
This `is_hash` check is not needed because we don't call `feed_loop` on atoms

Reply:
Nope... this check is performed on the children of `ptr`. This is an optimization I coded and then forgot 😮‍💨